crowbar_framework/app/models/backup.rb

```diff
@@ -23,6 +23,9 @@ class Backup < ActiveRecord::Base
   before_validation :save_or_create_archive, on: :create
   after_validation :delete_archive, unless: -> { errors.empty? }
   after_destroy :delete_archive
+  after_create do
```

Comment:
hmm this one is still nagging... @vuntz shall I drop it until we have a solution (we should at least track it in a card)? it shouldn't block the other fixes IMO